tests/Behat/CustomerContext/CustomerContext.php

```diff
@@ -268,11 +269,41 @@ public function getStatus(string $id): CustomerStatus
         return $status;
     }
 
+    /**
+     * @Given the customers database is empty
+     */
+    public function theCustomersDatabaseIsEmpty(): void
+    {
+        // Очищаємо всіх клієнтів з бази даних
```

Comment:
Please remove all comments, we are not using the Ukrainian language in opensource
Only English